uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestActivity.kt

```diff
@@ -324,7 +295,7 @@ class BookingRequestView @JvmOverloads constructor(context: Context,
     }
 
     override fun showPrebookConfirmationDialog(quoteType: QuoteType?, tripInfo: TripInfo) {
-        val activity = context as Activity
+        val activity = this as Activity
```

Comment:
there is no point in keeping this, it's was meant to go from context to activity, now it's useless

Reply:
Done